src/main/java/reproducer/Main.java

```diff
@@ -51,17 +52,21 @@ private static class Reproduce implements Runnable {
         @CommandLine.Option(
                 names = {"-f", "--file"},
                 paramLabel = "BREAKING-UPDATE-FILE",
-                description = "A JSON file describing a breaking update.",
-                required = true
+                description = "A JSON file describing a breaking update."
```

Comment:
A small suggestion to make the behavior more clear:
```suggestion
                description = "A JSON file for a specific breaking update to reproduce. If not provided, all breaking updates in the dataset directory which have not already been reproduced will be will be reproduced instead.
```